make/winx86/cmd/sh.cmd

```diff
@@ -1,135 +0,0 @@
-@echo off
-rem
-rem This file should be put into C:\Program Files\Git\cmd\ subdirectory
-rem (or similar, depeding on where the msysGit package was installed)
-rem to provide a shell prompt in the unix-like build environment on Windows.
-rem
-rem Currently supported on NT-class systems only (Windows XP and above).
-rem
-rem See also:
-rem   README.txt
-rem   http://wiki.openpilot.org/display/Doc/GCS+Development+on+Windows
-rem   http://wiki.openpilot.org/display/Doc/Firmware+Development+on+Windows
-rem
-rem Based on the msys.bat file from the MSYS package 
-rem   http://www.mingw.org/wiki/msys
-rem
-
-rem this should let run MSYS shell on x64
-if "%PROCESSOR_ARCHITECTURE%" == "AMD64" (
-  SET COMSPEC=%WINDIR%\SysWOW64\cmd.exe
-)
-
-rem some MSYS environment variables
-if "x%MSYSTEM%" == "x" set MSYSTEM=MINGW32
-if not "x%DISPLAY%" == "x" set DISPLAY=
-
-rem --------------------------------------------------------------------------
-rem To build the OpenPilot software we need few tools in the PATH.
-rem Here we attempt to guess tools location searching in the given
-rem directories first, and in the PATH last, if not found where expected.
-rem
-rem Please note that if you have few similar tools installed somewhere but
-rem not in the expected location, and they are in the PATH, then the script
-rem can detect wrong directories. For instance, if you have QtSDK installed
-rem on the D: drive, but have separately installed MinGW toolkit which is
-rem in the PATH, then this script detects this MinGW instead of QtSDK's one.
-rem As a result, the SDL headers will not be found, if they were copied into
-rem QtSDK's MinGW directory. In that case make sure that you have correct
-rem directories specified here.
-rem
-rem Also the SDL should be installed into Qt directories to build the GCS.
-rem Check the wiki or ground/openpilotgcs/copydata.pro for details.
-rem
-rem Also you can add any paths below just by adding extra 'call :which'
-rem lines with the following parameters:
-rem  - environment variable which will be set to the tool location, if found;
-rem  - expected directory for the executable which will be searched first;
-rem  - any executable file which will be searched in the given directory
-rem    or in the PATH, if not found where expected.
-rem All they will be added to the PATH in order of appearance.
-rem --------------------------------------------------------------------------
-
-set NOT_FOUND=
-set PATH_DIRS=
-
-if "%PROCESSOR_ARCHITECTURE%" == "AMD64" (
-  call :which MSYSGIT       "%ProgramFiles(x86)%\Git\bin" git.exe
-) else (
-  call :which MSYSGIT       "%ProgramFiles%\Git\bin"  git.exe
-)
-call :which QTMINGW       "C:\QtSDK\mingw\bin"                  mingw32-make.exe

```

Comment:
So glad to see all this go. This was always the confusing part of setting up a dev environment on Windows.